### DISCUSSION TITLE
Has anyone got Faikin running on an ESP32-S3-ZERO?

### DISCUSSION THREAD

**Discussion by TheBestMoshe** [revk/xESP32-Faikout#442] [Q&A]
I can flash it using this command

```
esptool.py write_flash 0x0 Faikin-S3-MINI-N4-R2-bootloader.bin 0x8000 partition-table.bin 0xd000 ota_data_initial.bin 0x10000 Faikin-S3-MINI-N4-R2.bin
```

However, it doesn't run. 

Checking the logs using `screen /dev/cu.usbmodem101 115200`, I see this error

```
E (1138) httpd: httpd_server_init: error in listen (112)
```

**Comment by revk** [comment]
You would only get that if it was running. Error in listen is odd, but may be Ok as it had not perhaps started the wifi AP yet.

Any other messages?

That build is for 4M flash and 2M SPIRAM

**Comment by TheBestMoshe** [comment]
I think there might be something wrong with the chip that I have. I flashed it with ESPHome and I also get a network error.
